lib/Common/Core/FaultInjection.cpp

```diff
@@ -150,7 +149,7 @@ namespace Js
             else
             {
                 RtlVirtualUnwind(UNW_FLAG_NHANDLER, ImageBase, Context.Rip, RuntimeFunction,
-                    &Context, &HandlerData, &EstablisherFrame, &NvContext);
+                    &Context, &HandlerData, &EstablisherFrame, NULL);
```

Comment:
Nitpicky but can you add a comment to the parameter indicating what the NULL is for- something like:
`NULL /* ContextPointers */`